C/charon/bundled/patches/charon-all-changes.patch.patch

```diff
@@ -0,0 +1,140 @@
+diff --git before/tcad-charon/PackagesList.cmake after/tcad-charon/PackagesList.cmake
+index 4a36e27..ffa9276 100644
+--- before/tcad-charon/PackagesList.cmake
++++ after/tcad-charon/PackagesList.cmake
+@@ -1 +1,3 @@
+-TRIBITS_REPOSITORY_DEFINE_PACKAGES()
++TRIBITS_REPOSITORY_DEFINE_PACKAGES(
++  Charon src SS
++)
+diff --git before/tcad-charon/TPLsList.cmake after/tcad-charon/TPLsList.cmake
+index 402fb98..43088ab 100644
+--- before/tcad-charon/TPLsList.cmake
++++ after/tcad-charon/TPLsList.cmake
+@@ -1 +1,25 @@
+-TRIBITS_REPOSITORY_DEFINE_TPLS()
++TRIBITS_REPOSITORY_DEFINE_TPLS(
++  Teuchos TRIBITS_PKG PT
++  Kokkos TRIBITS_PKG PT
++  Panzer TRIBITS_PKG PT
++  PanzerDiscFE TRIBITS_PKG PT
++  PanzerAdaptersSTK TRIBITS_PKG PT
++  STKMesh TRIBITS_PKG PT
++  STKUtil TRIBITS_PKG PT
++  STKTopology TRIBITS_PKG PT
++  SEACASNemesis TRIBITS_PKG PT
++  SEACASExodus TRIBITS_PKG PT
++  SEACASAprepro TRIBITS_PKG PT
++  Stratimikos TRIBITS_PKG PT
++  Piro TRIBITS_PKG PT
++  NOX TRIBITS_PKG PT
++  Rythmos TRIBITS_PKG PT
++  Teko TRIBITS_PKG PT
++  Belos TRIBITS_PKG PT
++  AztecOO TRIBITS_PKG PT
++  SEACAS TRIBITS_PKG PT
++  SEACASEpu TRIBITS_PKG PT
++  SEACASExodiff TRIBITS_PKG PT
++  SEACASNemslice TRIBITS_PKG PT
++  SEACASNemspread TRIBITS_PKG PT
++)
+diff --git before/tcad-charon/cmake/ProjectCompilerPostConfig.cmake after/tcad-charon/cmake/ProjectCompilerPostConfig.cmake
+index 35a1c23..7088f2c 100644
+--- before/tcad-charon/cmake/ProjectCompilerPostConfig.cmake
++++ after/tcad-charon/cmake/ProjectCompilerPostConfig.cmake
+@@ -1,2 +1 @@
+ SET(Trilinos_ENABLE_Kokkos ${${PROJECT_NAME}_ENABLE_Kokkos} CACHE BOOL "Setting Trilinos_ENABLE_Kokkos to ${PROJECT_NAME}_ENABLE_Kokkos in ProjectCompilerPostConfig.cmake.")
+-INCLUDE(${CMAKE_CURRENT_LIST_DIR}/../Trilinos/cmake/ProjectCompilerPostConfig.cmake)
+diff --git before/tcad-charon/src/CMakeLists.txt after/tcad-charon/src/CMakeLists.txt
+index 0ef0026..4d00bc9 100644
+--- before/tcad-charon/src/CMakeLists.txt
++++ after/tcad-charon/src/CMakeLists.txt
+@@ -88,7 +88,7 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
+   ${PACKAGE_NAME}_ENABLE_EXPLICIT_INSTANTIATION
+   HAVE_${PACKAGE_NAME_UC}_EXPLICIT_INSTANTIATION
+   "Enable the use of explicit template instantiation (defaults to Panzer value)."
+-  ${Panzer_ENABLE_EXPLICIT_INSTANTIATION}
++  OFF
```

Comment:
I'm not sure this is still required after I fixed the dependency listing above. Will have to try.